src/consensus/consensus.h

```diff
@@ -22,4 +34,94 @@ enum {
     LOCKTIME_MEDIAN_TIME_PAST = (1 << 1),
 };
 
+/**
+ * Context-independent CTransaction validity checks.
+ * Nobody should spend an extra cycle on a transaction that doesn't pass this.
```

Comment:
There's some other small documentation additions in consensus/consensus.h but they can all be postponed.
I'm happy to remove this doc line or any other as nit.

Regarding the withe space, the goal was to only change that kind of little style thing if it's to comply more with our style rules, but here I'm actually doing the opposite (https://github.com/bitcoin/bitcoin/blob/master/src/.clang-format#L41), so thank you for noticing on time. Probably lost in rebase at some point, I will change this.